### PR TITLE
Update: coachtopia brand logos added

### PR DESCRIPTION
**Pull request by ejaz-tapestry**

### CODE REVIEW

**Review by p-musi — APPROVED** [review]

**Review by p-musi — CHANGES_REQUESTED** [review]
@ejaz-tapestry, please review feedback.

File: src/brands/coachtopia/default/asset/logo.json [hunk 1]
```diff
@@ -1,21 +1,6 @@
...
-      },
-      "mark-white": {
-        "value": "mark-white.svg"
-      },
-      "monogram-black": {
-        "value": "monogram-black.svg"
-      },
-      "monogram-white": {
-        "value": "monogram-white.svg"
-      },
```
[the remark this answers is not in the document]
Comment (ejaz-tapestry) [reply src/brands/coachtopia/default/asset/logo.json]: @p-musi code added back

**Review by p-musi — APPROVED** [review]

File: src/brands/coachtopia/default/effect/box-shadow.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "effect": {
+    "box-shadow": {
+      "ltr": {
+        "value": "0px -5px 10px -1px rgba(0, 0, 0, 0.05),\n-6px 0px 10px -5px rgba(0, 0, 0, 0.05),  \n6px 0px 10px -5px rgba(0, 0, 0, 0.05)"
+      },
```
Comment (aoverstreet-tapestry) [inline src/brands/coachtopia/default/effect/box-shadow.json:6]: @ejaz-tapestry Please remove "ltr" from this file.  
However, please update the "ltr" in global/effect/box-shadow.json with the follow:
"value": "0px -5px 10px -1px rgba(0, 0, 0, 0.05), -6px 0px 10px -5px rgba(0, 0, 0, 0.05), 6px 0px 10px -5px rgba(0, 0, 0, 0.05)"

Comment (ejaz-tapestry) [reply src/brands/coachtopia/default/effect/box-shadow.json:6]: ltr added to global setttings.

**Review by p-musi — APPROVED** [review]

**Review by aoverstreet-tapestry — APPROVED** [review]

**Pull request merged**